includes/template-functions.php

```diff
@@ -26,6 +26,17 @@ function sensei_course_archive_next_link( $type = 'newcourses' ) {
 	  * @return void
 	  */
 function course_single_lessons() {
+	/**
+	 * Access check for the course lessons.
+	 *
+	 * @since 2.2.0
+	 *
+	 * @param bool $show_lessons Whether or not the lessons should be shown. Default true.
+	 * @param int  $course_id    Course ID.
+	 */
+	if ( ! apply_filters( 'sensei_course_lessons_has_access', true, get_the_ID() ) ) {
```

Comment:
See above comment re: copying the filter and documentation.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/2763/commits/14e2809502dacf0478bdb18585a221f60c7806aa.